### DISCUSSION TITLE
Welcome to v2rayNG Discussions!

### DISCUSSION THREAD

**Discussion by 2dust** [2dust/v2rayNG#954] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rurirei** [comment]
am i the first one? 😄

**Comment by naima420** [comment]
嗯

**Comment by tyronesaint** [comment]
Version 1.6.3 of v2rayng, v2ray tcp with obfuscation have been unable to connect, showing read/write on closed pipe

**Comment by qidor** [comment]
最新版的4.13怎么用全局啊，用IP测试环境一直是本地网络。

**Comment by zhou431615** [comment]
好期待Ghelper3客户端的发布！是6月份发布发布吗？

**Comment by ntofound4041202** [comment]
Bagaimana cara hospot tetring ke andorid lain?

**Reply by miguel4tech** [reply]
Please,. I need help with this as well, I don't think the VPN is being share over hotspot from Android to Android

**Reply by brian12392** [reply]
Hello, 
VPN shared over hotspot? 
what kind of device are  you talking about

**Comment by brian12392** [comment]
Hello , good to be here 
Thanks for developing such a good app ,
I always use this app for my business in china 

But lately I find that I can not find your app in the galaxy wearable app store , I use Galaxy watch 3 with Esim service , so it's kinda just like a mobile phone because with Esim service, the galaxy smart watch is connected to mobile internet data,
Here comes the question , can you please develop a V2rayNg  app for the Galaxy watch 3? So that when Esim service is used with the watch , the watch can connect to free internet  in China

**Comment by miguel4tech** [comment]
A Samsung  A12 android device with the latest OS 11.Running on a Nigerian network,  Glo Ng 4G

Sent from Yahoo Mail on Android 
 
  On Mon, Jun 21, 2021 at 2:33 AM, ***@***.***> wrote:   


Hello,
VPN shared over hotspot?
what kind of device are you talking about

**Comment by mmofaha** [comment]
I like it v2rayNG. It run smoothly.
Can u please add a menu of "REORDER BY DELAY" so easily to select which proxy is best conncection at that time.
SagerNet and AnXray have this menu inside it.
TQVM

**Comment by yan2035** [comment]
三星国行上不去YouTube客户端和谷歌商店，这问题有解么。网页都正常，就是app上不去

**Comment by brian12392** [comment]
三星国行的 GALAXY WATCH 3 和美版的我都测试过了，结果就是如果要用YouTube 或者谷歌（谷歌助手）国行基本不能用 ，很不稳定，
美版的没问题，
